CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+## [4.1.0] - 2023-11-07
+
+- Add a new "Optimized prepared statement" option when defining a database
```

Comment:
You put it under [Unreleased] and it will be automatically moved to the correct version when you do the release.

Reply:
Updated